### ISSUE TITLE
init weights from pre_train model resnet_v2_101 something wrong :    Assign requires shapes of both tensors to match. lhs shape= [19] rhs shape= [256]

### ISSUE THREAD

**Issue by Suodislie** [tensorflow/tensorflow#12870]
**SSD generate feature map by VGG16, I replaced VGG16 with ResNet_V2_101,the code as follows:**
`  with slim.arg_scope(resnet_utils.resnet_arg_scope()):
         net , end_points =resnet_v2.resnet_v2_101(inputs,reuse=reuse,global_pool=False,is_training=True,scope='resnet_v2_101')
`
           
**when i restore the pretrain_weights from the checkpoint_path, something wrong as follows:**
`InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [19] rhs shape= [256]
	 [[Node: save_1/Assign_294 = Assign[T=DT_FLOAT, _class=["loc:@resnet_v2_101/block3/unit_18/bottleneck_v2/conv2/BatchNorm/gamma"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/cpu:0"](resnet_v2_101/block3/unit_18/bottleneck_v2/conv2/BatchNorm/gamma, save_1/RestoreV2_294)]]
`
**## I do not konw why, Hope for your help!**

**Comment by cy89** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
